.github/workflows/test-template.yml

```
@@ -0,0 +1,250 @@
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install dependencies
+        run: |
+          uv sync --all-extras
+
+      - name: Run generation tests for ${{ matrix.config }}
+        run: |
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2102:info:2:93: Ranges can only match single chars (mentioned due to duplicates)

(shellcheck)